cranelift/codegen/src/isa/x64/lower.rs

```diff
@@ -1033,21 +1033,11 @@ fn lower_insn_to_regs<C: LowerCtx<I = Inst>>(
         }
 
         Opcode::Fdemote => {
-            // We can't guarantee the RHS (if a load) is 128-bit aligned, so we
-            // must avoid merging a load here.
-            let src = RegMem::reg(put_input_in_reg(ctx, inputs[0]));
-            let dst = get_output_reg(ctx, outputs[0]).only_reg().unwrap();
-            ctx.emit(Inst::xmm_unary_rm_r(SseOpcode::Cvtsd2ss, src, dst));
+            implemented_in_isle(ctx);
         }
 
         Opcode::Fvdemote => {
-            let src = RegMem::reg(put_input_in_reg(ctx, inputs[0]));
-            let dst = get_output_reg(ctx, outputs[0]).only_reg().unwrap();
-            ctx.emit(Inst::xmm_unary_rm_r(
-                SseOpcode::Cvtpd2ps,
-                RegMem::from(src),
-                dst,
-            ));
+            implemented_in_isle(ctx);
```

Comment:
Ideally we would just move these opcodes up to the top of the match where the rest of the implemented-in-ISLE opcodes are all matched together.

Reply:
I'll move them up to the top -- I was matching the surrounding style, figuring that we'd do a pass over this later.